go.mod

```diff
@@ -159,24 +159,24 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 	gorm.io/driver/postgres v1.5.11
 	gorm.io/gorm v1.25.10
-	helm.sh/helm/v3 v3.15.4
-	k8s.io/api v0.30.9
-	k8s.io/apiextensions-apiserver v0.30.9
-	k8s.io/apimachinery v0.30.9
-	k8s.io/apiserver v0.30.9
-	k8s.io/cli-runtime v0.30.9
-	k8s.io/client-go v0.30.9
-	k8s.io/kubectl v0.30.9
+	helm.sh/helm/v3 v3.18.6
+	k8s.io/api v0.33.3
+	k8s.io/apiextensions-apiserver v0.33.3
+	k8s.io/apimachinery v0.33.3
+	k8s.io/apiserver v0.33.3
+	k8s.io/cli-runtime v0.33.3
+	k8s.io/client-go v0.33.3
+	k8s.io/kubectl v0.33.3
```

Comment:
I'm not sure bump of helm where the vulnerable code is unused justifies the risk of bumping past 3 releases of k8s 🤔

Reply:
I was not sure as well, I wanted to try and see what CI says.

Since it's not looking good I would also rather close this PR again.